VE-2084: Direct user to correct page after clicking yes

pauloslund opened the pull request.

Commit:
VE-2084: Direct user to correct page after clicking yes

Commit:
VE-2084: Cleanup JS style issues

inez commented on extensions/wikia/EmergencyBroadcastSystem/js/EmergencyBroadcastSystem.js:
```diff
@@ -1,11 +1,22 @@
-})(jQuery);
+	$( '.ebs-primary-action' ).click( function(){
+		var href = this.href;
+		var request = handleClick( 1 );
```
```
handleClick( 1 ).then( function() {
...
}
```

That way you don't not the request variable.
Btw. using then is sort of better - because it does the same thing (more or less) but it is a promise standard - according to spec.

inez replied:
Also if you bind then you wouldn't have to ever have href variable - up to you.

Commit:
VE-2084: Switch to using promise based callbacks

Commit:
VE-2084: Rename methdo handleClick to more appropriate saveUserResponse

Also don't preventDefault/return false in click handler for clicking no - which should take user to survey
